.github/workflows/main.yaml

```diff
@@ -0,0 +1,28 @@
+name: Main
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - 'docs/**'
+      - '**/*.md'
+      - 'tests/performance/**'
```

Comment:
Maybe we should also cache on changes in perf tests
```suggestion
```

Reply:
Since this only applies to caching Go dependencies, I don't see a benefit of running this on performance test changes as well, or am I missing something?